src/test/java/org/minions/devfund/noemiguzman/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,148 @@
+package org.minions.devfund.noemiguzman.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *  Class to test movie database.
+ */
+public class MovieDatabaseTest {
+    private MovieDatabase moviedb1;
+    private MovieDatabase moviedb2;
+
+    private String actorBale = "Christian Bale";
+    private String actorWest = "Adam West";
+    private String actorAffleck = "Ben Affleck";
+    private String movieGlass = "Glass";
+    private String movieBatman = "Batman";
+    private String movieIt = "It";
+    private Movie movie1;
+
+    private List<Movie> movieList1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "movieList1" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)